system/settings/settings.xml

```diff
@@ -3371,8 +3371,8 @@
           <default>0</default>
           <constraints>
             <minimum label="351">0</minimum>
-            <step>5</step>
-            <maximum>120</maximum>
+            <step>15</step>
+            <maximum>240</maximum>
```

Comment:
might as well make this something large like 24 hours. It may be better to use a different control type as a spinner would be annoying for large values (IMO).

Reply:
maybe something like 

```
--- a/system/settings/settings.xml
+++ b/system/settings/settings.xml
@@ -3390,11 +3390,11 @@
           <level>2</level>
           <default>0</default>
           <constraints>
-            <minimum label="351">0</minimum>
-            <step>5</step>
-            <maximum>120</maximum>
+            <minimum>0</minimum>
+            <step>15</step>
+            <maximum>1440</maximum>
           </constraints>
-          <control type="spinner" format="string">
+          <control type="slider" format="integer">
             <formatlabel>14044</formatlabel>
           </control>
         </setting>
```

<img width="1131" alt="Screenshot 2023-06-05 at 10 14 21 am" src="https://github.com/xbmc/xbmc/assets/2861319/31afdb2c-8716-4cf3-8182-dca29e90097e">

Reply:
Updated patch but haven't tested to check whether control type slider would show this as off when at 0 like spinner does?